src/libraries/System.Runtime.InteropServices.JavaScript/tests/AssemblyInfo.cs

```diff
@@ -0,0 +1,7 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Xunit;
+
+[assembly: ActiveIssue("https://github.com/dotnet/runtime/issues/34748", TestPlatforms.Windows, TargetFrameworkMonikers.Netcoreapp, TestRuntimes.Mono)]
```

Comment:
This issue is about COM tests failing on Mono. Copy & paste mistake?

Reply:
Oh definitely copy paste error.  Thought I had that changed already.